### PR TITLE
Language Config

### PR DESCRIPTION
**Pull request by aannyyaa**

### CODE REVIEW

File: src/components/footer.jsx [hunk 1]
```diff
@@ -18,8 +18,8 @@ function Footer() {
 
   const primaryLinks = links.primary;
   const primaryLinksLength = primaryLinks.length;
-  const primaryLinksLabels = t('navigation.dropdowns').map((element) => element.linkLabels).flat().concat(t('navigation.parentLinkLabels'));
-  const primaryLinksLabelsLength = primaryLinksLabels.length;
+  const primaryLinksLabels = t('navigation.dropdowns').map && t('navigation.dropdowns').map((element) => element.linkLabels).flat().concat(t('navigation.parentLinkLabels'));
```
Comment (saifulislamdev) [inline src/components/footer.jsx]: The logic in this file has changed through my PRs. I would recommend merging dev into this branch and making sure what you have here works.

File: src/components/footer.jsx [hunk 2]
```diff
@@ -18,8 +18,8 @@ function Footer() {
...
+  const primaryLinksLabelsLength = primaryLinksLabels?.length;
```
Comment (saifulislamdev) [inline src/components/footer.jsx:26]: Interesting, I've never seen this notation before.

File: src/components/i18n.js [hunk 3]
```diff
@@ -1,65 +0,0 @@
-import i18n from 'i18next';
```
Comment (saifulislamdev) [inline src/components/i18n.js:1]: I made changes in this file that are available in dev. No major changes, but having those changes in your new file would be great.

File: src/components/nav-drop-down.jsx [hunk 4]
```diff
@@ -37,7 +37,7 @@ function NavDropDown() {
       const navDropDownLinks = dropDownLinks[i];
       const navDropDownLinksLength = navDropDownLinks.length;
       const navDropDownLinkLabels = dropDownLabels[i].linkLabels;
-      const navDropDownLinkLabelsLength = navDropDownLinkLabels.length;
+      const navDropDownLinkLabelsLength = navDropDownLinkLabels?.length;
```
Comment (saifulislamdev) [inline src/components/nav-drop-down.jsx]: The logic in this file has also changed through my PRs. I would recommend merging dev into this branch and making sure what you have here works.

File: src/i18n-config.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
+import i18n from 'i18next';
```
Comment (saifulislamdev) [inline src/i18n-config.js:2]: How come this file is in `src/` and not `src/components/`?

Comment (aannyyaa) [reply src/i18n-config.js:2]: Since it's not a component I moved it to src

File: src/components/i18n.js (hunk 3 above)
Comment (aannyyaa) [reply src/components/i18n.js:1]: I think they're all incorporated

Comment (saifulislamdev) [reply src/components/i18n.js:1]: Even the comment?

Comment (aannyyaa) [reply src/components/i18n.js:1]: Yeah I copied and pasted the comment over when resolving the merge conflicts

**Pull request merged**